lib/buildingsync/model_articulation/loads_system.rb

```diff
@@ -149,26 +149,59 @@ def get_building_section(building_sections, standard_building_type, standard_spa
       return nil
     end
 
-    def adjust_people_schedule(space_type, building_section, model)
-      if !building_section.typical_occupant_usage_value_hours.nil?
-        puts_string = "building_section.typical_occupant_usage_value_hours: #{building_section.typical_occupant_usage_value_hours}"
+    def adjust_schedules(standard, space_type, building_section, model)
```

Comment:
@TMaile is this a `building_section` or more at the `building` level?  Please change parameter name to `building` if so

Reply:
this is addressed in https://github.com/BuildingSync/BuildingSync-gem/commit/1d14b861aad9edc057934b64af8d45f50ee45cd9
taking the average value from the sections at the building level